main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"path"
 	"strconv"
 
-	"github.com/codegangsta/cli"
+	"github.com/docker/machine/cli"
```

Comment:
https://github.com/codegangsta/cli/pull/278

I do wish I could leave it out of the diff in this PR, though :(